README.rst

```diff
@@ -83,7 +83,7 @@ or have a look at
 .. |azure_build| image:: https://dev.azure.com/gamma-facet/facet/_apis/build/status/BCG-Gamma.pytools?branchName=develop
    :target: https://dev.azure.com/gamma-facet/facet/_build?definitionId=9&_a=summary
 
-.. |azure_code_cov| image:: https://img.shields.io/azure-devops/coverage/gamma-facet/facet/_build?definitionId=9.svg
+.. |azure_code_cov| image:: https://img.shields.io/azure-devops/coverage/gamma-facet/facet/9/develop.svg
```

Comment:
Do we want the badge for facet?

Reply:
Update: It's fine that way for pytools, project name in URI is facet, pytools build id is 9

Reply:
the id value of 9 before `/develop` should point to pytools in Devops.